src/core/folder.h

```diff
@@ -179,6 +183,8 @@ private Q_SLOTS:
     bool defer_content_test : 1;
 
     static std::unordered_map<FilePath, std::weak_ptr<Folder>, FilePathHash> cache_;
+    static FilePath cutFilesDirPath_;
+    static std::shared_ptr<const HashSet> cutFilesHashSet_;
```

Comment:
> I think I've found a solution...

It's always enough to just report a bug. I'm here to support (especially) my code.